tests/drivers/flash/testcase.yaml

```diff
@@ -11,3 +11,7 @@ tests:
     platform_allow: nrf52840dk_nrf52840
     tags: nrf52 soc_flash_nrf
     extra_args: OVERLAY_CONFIG=boards/nrf52840_flash_soc.conf
+  drivers.flash.soc_nxp_rt1xxx:
+    platform_allow: mimxrt1060_evk
+    tags: mcux soc_rt1xxxx
+    extra_args: OVERLAY_CONFIG=boards/mimxrt1060_evk.conf
```

Comment:
The `extra_args` shouldn't be necessary because the build system automatically looks for a board-specific overlay. If you remove `extra_args`, you could then rename the test to something more generic, like `drivers.flash.default`

Reply:
the problem is that the test cases yaml is still wring in old format. see below
```
tests:
  drivers.flash.nrf_qspi_nor:
    platform_allow: nrf52840dk_nrf52840
    tags: flash nrf52 nrf_qspi_fash
    extra_args: OVERLAY_CONFIG=boards/nrf52840_flash_qspi.conf
  drivers.flash.soc_flash_nrf:
    platform_allow: nrf52840dk_nrf52840
    tags: nrf52 soc_flash_nrf
    extra_args: OVERLAY_CONFIG=boards/nrf52840_flash_soc.conf
```
there is no default ones, only mimxrt1060_evk and 2 nodic boards supports this case.